opal/templates/patient_lists/partials/list_dropdown_menu.html

```diff
@@ -1,21 +1,26 @@
 {% load forms %}
-{% if num_lists > 1 %}
+
+<!-- suppress the Switcher if there are no items in it -->
+{% if switcher_items != None %}
```

Comment:
Does this work ?

I _think_ that python truthines would work on `{% if switcher_items %}` ([] evaluates to bool False) but it can't see a path for it to be None from the view ?